datalad/interface/run_procedure.py

```diff
@@ -151,6 +199,25 @@ class RunProcedure(Interface):
     of taking at least one positional argument (the absolute path to the
     dataset they shall operate on).
 
+    For further customization there are two configuration settings per procedure
+    available:
+
+    - 'datalad.procedures.<NAME>.call-format'
+      fully customizable format string to determine how to execute procedure
+      NAME (see also datalad-run).
```

Comment:
I'm not sure we should refer to `run` here because I don't imagine we want to encourage passing through placeholders like "{{pwd}}" in the call format.  The view the fact that `run-procedure` uses `run` underneath as an implementation detail.

Reply:
Never mind.  I see that that one of the tests explicitly deals with passing through substitution placeholders, so I will update my view that using `run` is an implementation detail :]

Reply:
:)
Yes. And this passing through of substitutions is a main point of the toolbox-usecase that led to coming up with this PR.

Reply:
I don't understand that, but I take your word for it.